pkg/asset/manifests/operators.go

```diff
@@ -74,13 +74,12 @@ func (m *Manifests) Dependencies() []asset.Asset {
 		&bootkube.HostEtcdServiceEndpointsKubeSystem{},
 		&bootkube.HostEtcdServiceKubeSystem{},
 		&bootkube.KubeCloudConfig{},
-		&bootkube.KubeSystemConfigmapEtcdCA{},
+		&bootkube.OpenshiftConfigConfigmapEtcdCA{},
```

Comment:
We're not super-consistent about this, but personally I'd rather have `OpenShiftConfigConfigmapEtcdCA` (why break with our canonical product casing?).  Or, drop the namespace entirely, since `bootkube.EtcdCAConfigMap` seems like plenty of name to me on its own.